images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -35,40 +84,10 @@ pacman.exe -Syuu  --noconfirm
 taskkill /f /fi "MODULES eq msys-2.0.dll"
```

Comment:
Do we want to do all these steps in case we don't install any packages?

Reply:
I am not sure about exact purpose of these lines so decided to keep them. Do you have any context?
May be @al-cheb knows

Reply:
Per discussion with @al-cheb , these lines should be run even if we don't install packages.